core/jobs_test.py

```diff
@@ -806,6 +806,10 @@ def test_raise_error_with_get_continuous_computation_class(self):
                 'BaseMapReduceJobManagerForContinuousComputations must '
                 'implement the _get_continuous_computation_class() method.')):
             (
+                # Access of private attribute OK because it is testing
```

Comment:
Please write using complete sentences. E.g.:

"The private method is being used here because we are testing the structure of an abstract base class (in particular, that one of its methods is defined and raises NotImplementedError), and there is no other way to access this method."